generator/src/main/java/com/algorand/sdkutils/listeners/GoGenerator.java

```diff
@@ -0,0 +1,678 @@
+        try {
+            bw.close();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    String goType(String type, boolean array) {
+        return goType(type, array, false, "").type;
+    }
+
+    class TypeConverter {
+        public String type;
+        public String converter;
+
+        public TypeConverter(String type, String converter) {
+            this.type = type;
+            this.converter = converter;
+        }
+    }
+
+    TypeConverter goType(String type, boolean array, boolean asType, String paramName) {
+
+        String goType = "";
+        String converter = paramName;
+
+        switch (type) {
+        case "boolean":
+            goType = "bool";
+            break;
+        case "integer":
+            goType = "uint64";
+            break;
+        case "string":
+            goType = type;
+            break;        
+        case "Account":
+            goType = "string";
+            break;
+        case "address":
+            goType = asType ? "types.Address" : "string";
+            if (asType) {
+                addImport("C", "github.com/algorand/go-algorand-sdk/types");
+                converter = paramName + ".String()";
+            }
+            break;
+
+        case "time":
+            goType = asType ? "time.Time" : "string";
+            if (asType) {
+                addImport("A", "time");
+                converter = paramName + ".Format(time.RFC3339)"; 
+            }
+            break;
+
+        case "byteArray":
+            goType = asType ? "[]byte" : "string";
+            if (asType) {
+                addImport("A", "encoding/base64");
+                converter = "base64.StdEncoding.EncodeToString(" + paramName +")";
+            }
+            break;
+        case "object":
+            goType =  "*map[string]interface{}";
+            break;
+            
+        //case "array":
+          //  goType = 
+
+        case "Asset":
+        case "AssetHolding":
+        case "AssetParams":
+        case "AccountParticipation":
+        case "Application":
+        case "ApplicationLocalState":
+        case "ApplicationStateSchema":
+        case "ApplicationParams":
+        case "BlockRewards":
+        case "BlockUpgradeState":
+        case "BlockUpgradeVote":
+        case "MiniAssetHolding":
+        case "TealValue":
+        case "TealKeyValue":
+        case "Transaction":
+        case "TransactionAssetConfig":
+        case "TransactionAssetFreeze":
+        case "TransactionAssetTransfer":
+        case "TransactionKeyreg":
+        case "TransactionPayment":
+        case "TransactionSignature":
+        case "TransactionSignatureLogicsig":
+        case "TransactionSignatureMultisig":
+        case "TransactionSignatureMultisigSubsignature":
+            goType = type;
+            break;
+        default:
+            goType = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx";            
```

Comment:
Is this actually hit? If it's an error it should be an error.

Reply:
Yes, this needs to throw an exception. However, it is convenient to know what is missing by just looking into the generated code, without going to the debugger.